### PR TITLE
[SMALLFIX] Fix documents for Tachyon Configuration and YARN integration

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/Running-Tachyon-on-EC2-Yarn.md [hunk 1]
```diff
@@ -146,52 +147,89 @@ master/workers, put `tachyon-site.properties` under `${TACHYON_HOME}/conf` on ea
...
+```
+
 You can run tests against Tachyon to check its health:
 
 ```bash
 $ /tachyon/bin/tachyon runTests
 ```
 
-After the tests finish, visit Tachyon web UI at `http://{MASTER_IP}:19999` again. Click `Browse
+After the tests finish, visit Tachyon web UI at `http://{Your_Master_IP}:19999` again. Click `Browse
```
Comment (jsimsa) [inline docs/Running-Tachyon-on-EC2-Yarn.md]: `TACHYON_MASTER_IP` instead of `Your_Master_IP`

File: docs/Running-Tachyon-on-EC2-Yarn.md [hunk 2]
```diff
@@ -146,52 +147,89 @@ master/workers, put `tachyon-site.properties` under `${TACHYON_HOME}/conf` on ea
...
 File System` in the navigation bar, and you should see the files written to Tachyon by the above
 tests.
 
 
 # Stop Tachyon
 
-Tachyon can be stopped by using the following Yarn command where the application ID of Tachyon can
-be retrieved from either Yarn web UI or the output of `tachyon-yarn.sh`:
+Tachyon can be stopped by using the following YARN command where the application ID of Tachyon can
+be retrieved from either YARN web UI or the output of `tachyon-yarn.sh` as mentioned above. In
```
Comment (jsimsa) [inline docs/Running-Tachyon-on-EC2-Yarn.md]: Replace `In our case ...` with `For instance, if the application ID is ..., you can stop Tachyon by killing the application using:`

**Comment by jsimsa** [comment]
Left some minor comments for you.

File: docs/Running-Tachyon-on-EC2-Yarn.md (hunk 1 above)
Comment (apc999) [inline docs/Running-Tachyon-on-EC2-Yarn.md]: done

File: docs/Running-Tachyon-on-EC2-Yarn.md (hunk 2 above)
Comment (apc999) [inline docs/Running-Tachyon-on-EC2-Yarn.md]: done

Comment (jsimsa) [inline docs/Running-Tachyon-on-EC2-Yarn.md]: For consistency, it might make sense to use `APPLICATION_ID` instead of `application_1445469376652_0002`

**Comment by calvinjia** [comment]
@apc999 It seems like the latest changes have not been pushed?

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**